docs/Interaction_Guidelines/ux_build_test.md

```diff
@@ -23,42 +19,24 @@ Once you're ready to go, you can start building and customizing your extension.
 
 * [Community Forums](https://community.tableau.com/community/developers/extensions-api/overview)<br>Participate in a community of developers that are passionate about creating extensions.
 
-* [Extension Gallery (beta)](https://extensiongallery.tableau.com/)<br>Explore and use some of the extensions our partners have created.
+* [Tableau Exchange](https://exchange.tableau.com/)<br>Explore and use some of the extensions our partners have created.
```

Comment:
Thanks for catching all these gallery references. I missed them completely when I updated references earlier.